Parse/src/main/java/com/parse/ParseObject.java

```diff
@@ -4237,7 +4246,26 @@ public void writeToParcel(Parcel dest, int flags) {
 
   /* package */ void writeToParcel(Parcel dest, ParseParcelEncoder encoder) {
     synchronized (mutex) {
-      // Write className and id regardless of state.
+      // Developer warnings.
+      ldsEnabledWhenParceling = Parse.isLocalDatastoreEnabled();
+      boolean saving = hasOutstandingOperations();
+      boolean deleting = isDeleting || isDeletingEventually > 0;
+      if (saving) {
+        Log.w(TAG, "About to parcel a ParseObject while a save / saveEventually operation is " +
```

Comment:
Will it be easy to trigger these issues if people are navigating between activities or you have a lot of rotation-type changes?